cmd/tests/cmd_run_test.go

```diff
@@ -2300,12 +2300,12 @@ func TestBrowserPermissions(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
 			script := fmt.Sprintf(`
-			import { browser } from 'k6/experimental/browser';
+			import { browser } from 'k6/browser';
```

Comment:
I think we should have a test for both imports until we remove the experimental import.

Reply:
This test is about k6 options and is not relevant to the module path.

In the commit, I changed the import path just to accommodate the module's current state. So, I don't think we should add another test here.

Reply:
Ok, got it, so it's nothing to do with the import itself.

I still think it's worth duplicating this test and the browser.js test, just to be extra cautious and ensure that both imports still work with what we're testing here.

Reply:
Duplicating them won't be worth it because we already tested the `browser` option. Instead, I've added a test that checks whether the `k6/experimental/browser` path is there and gives us a warning.

See: [48683b0](https://github.com/grafana/k6/pull/3793/commits/48683b09fdfcd60c5e2fe3cc2b04143e83993ffd)